src/code/migrations/02_convert_gems_to_array.js

```diff
@@ -0,0 +1,26 @@
+const version = 2;
+
+export default function update(state) {
+  if (state.stateVersion < version) {
+    state.stateVersion = version;
+    return doUpdate(state);
+  }
+  return state;
+}
+
+// Previously: each mission had one number representing the last gem awarded.
+// Now: each mission has an array of all gems awarded for that mission.
+function doUpdate(state) {
```

Comment:
Does this migration leave behind the old `stateVersion: 1` on stored data?

Reply:
On line 5 it updates the version in the state. Does that answer the question? Would it make more sense to you to put that in `index.js`?